README.md

```diff
@@ -1,2 +1,170 @@
-# Njoroge-Portfolio-Website
-First professional Portfolio Website
+<a name="readme-top"></a>
+
+<div align="center">
+  <!-- <img src="murple_logo.png" alt="logo" width="140"  height="auto" /> -->
+  <br/>
+
+  <h3><b>Portfolio Project</b></h3>
+
+</div>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+
+# 📖 Portfolio Project <a name="about-project"></a>
+
+**My portfolio project** is a website dedicated to summarizing my profile as a software developer. 
+It is also plays a crucial role on showcasing my abilties as a web developer.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  
+</details>
+
+<details>
+  <summary>Server</summary>
+  
+</details>
+
+<details>
+<summary>Database</summary>
+  
+</details>
+
+## Key Features <a name="key-features"></a>
+
+- **Header/Navbar(Provides an easy way to browse though the site)**
+- **Projects(Showcases different projects authored by me)**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://francis450.github.io/Njoroge)
```

Comment:
- [ ] Please make sure the Readme is professional and there is no broken link present in it, currently your Live demo link is broken and is throwing a 404 error, please fix this